nautobot/ui/src/components/Retrieve/RenderTable.js

```diff
@@ -1,7 +1,6 @@
 import { Table, Tbody, Td, Tr } from "@nautobot/nautobot-ui";
 
 import { ObjectTableItem } from "@components";
-import { buildUrl } from "@utils/url";
```

Comment:
Is this buildUrl()` function still needed? This appears to be the only place it was used. If not, please go ahead and delete it, too!

Reply:
Yeetus Deletus